packages/react-jss/src/index.js

```diff
@@ -3,4 +3,5 @@ export {ThemeProvider, withTheme, createTheming} from 'theming'
 export {default as JssProvider} from './JssProvider'
 export {default as jss} from './jss'
 export {SheetsRegistry, createGenerateId} from 'jss'
+export {default as __Context} from './JssContext'
```

Comment:
why not export as  JssContext?

Reply:
It shouldn't be used by the public. `react-router` also exports their context as `__Context`. 

I just looked at `react-redux` and they also just export their context.

Reply:
I think we should not document it, thats it.